internal/web/response.go

```diff
@@ -46,3 +46,18 @@ func (s *server) swappingRedirect(w http.ResponseWriter, path string, target str
 	locationJson, _ := json.Marshal(locationData)
 	w.Header().Add("Hx-Location", string(locationJson))
 }
+
+func (s *server) sendYamlModal(w http.ResponseWriter, obj string, err error) {
```

Comment:
If I understand correctly, `err` here does _not_ represent an error that happened during YAML marshalling, correct? If that is the case, I think this should be made more explicit somehow because initially that's what I assumed this parameter was for.